### PR TITLE
PROGRAMMES-6564 - Fix tab order

### PR DESCRIPTION
**Pull request by Bobkidbob**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6564

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/DsShared/Utilities/EntityContext/entity_context.html.twig [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {%- for item in entity_context.getAncestry() -%}
-    <a tabindex="-1" class="context__item" href="{{ path('find_by_pid', {'pid': item.getPid()}) }}">{{ item.getTitle() }}</a>
+    <a class="context__item" href="{{ path('find_by_pid', {'pid': item.getPid()}) }}"
```
Comment (BjornTwachtmann) [inline src/DsShared/Utilities/EntityContext/entity_context.html.twig]: The problem with this is that it's not going to differentiate between making visually hidden links tabbable and visible links tabbable, is it?

E.g. the hidden "doctor who" link here http://sandbox.bbc.co.uk/programmes/b09kkt1m will still get tabbed to? Is that what we want?

Comment (BjornTwachtmann) [reply src/DsShared/Utilities/EntityContext/entity_context.html.twig]: The first context__item always seems to be hidden by the CSS

**Comment by Bobkidbob** [comment]
Test this please.

**Pull request merged**